packages/fast-components-styles-msft/src/utilities/acrylic.ts

```diff
@@ -0,0 +1,28 @@
+import { applyAcrylic, IAcrylicConfig, toPx } from "@microsoft/fast-jss-utilities";
+import { ICSSRules } from "@microsoft/fast-jss-manager";
+import designSystemDefaults, { IDesignSystem } from "../design-system";
+import Chroma from "chroma-js";
```

Comment:
Were we going to use fast-colors instead of this?

Reply:
Is fast colors going to replace Chroma? @nicholasrice